app/src/main/java/com/simplecity/amp_library/playback/PlaybackManager.java

```diff
@@ -548,8 +548,10 @@ public void play() {
     }
 
      void newPlayerEvent(Song song, PlayerEventType playerEventType){
-        PlayerEvent playerEvent = EventUtils.newPlayerEvent(song, playerEventType, ShuttleApplication.get());
-        FirebaseIOUtils.savePlayerEvent(playerEvent);
+        if (song != null) {
```

Comment:
I'd remove the null check here because we want to know if we're missing logging events due to null songs. If it crashes, we'll know via the Google Play console and then we can fix it.